app/models/page.rb

```diff
@@ -2,7 +2,7 @@ class Page < ApplicationRecord
   belongs_to :form
   acts_as_list scope: :form
 
-  ANSWER_TYPES = %w[single_line number address date email national_insurance_number phone_number long_text selection organisation_name text].freeze
+  ANSWER_TYPES = %w[single_line number address date email national_insurance_number phone_number long_text selection organisation_name text name].freeze
```

Comment:
Shouldn't there be some tests around these new types?

Reply:
@aliuk2012 Think this is already covered - this `ANSWER_TYPES` constant is sampled in the page factory so the new answer types will be tested in the page model tests